bin/space2underscore

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env ruby
-
 require 'space2underscore'
 
-result  = Space2underscore.convert(ARGV)
-puts Space2underscore.create_new_branch(result)
+args = ARGV.reject { |arg| /-c|--create/ === arg }
```

Comment:
This will anything containing `-c`, so `/-c|--create/ === my-case` will return true.  Anyway, I think `%w(-c --create).include?(arg)` is simpler than a regexp here.